hack/cleanup-hypershift-leaks.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+export NOW=$(date +%s)
+echo Cleanning up resources older than $(date -u -d @$NOW +%Y-%m-%dT%H:%M:%SZ)
+
+echo 'Deleting expired OpenID Connect Providers...'
+# Get all OpenID Connect Providers with ARN
+arns=`aws iam list-open-id-connect-providers --query 'OpenIDConnectProviderList[*].Arn' --output json | jq -r '.[]'`
```

Comment:
what other tools are needed as dependencies of this script? I would mark that somehow.